.github/workflows/ci.yaml

```diff
@@ -35,14 +35,12 @@ jobs:
           # The small build we'll use as an integration test case.
           git clone https://github.com/nextstrain/zika-tutorial
 
-      - name: Check python version
-        run: |
-          # Assert that we're on the expected Python version, in case the GH
-          # Actions environment is messed up.
-          python --version | grep -F 'Python ${{ matrix.python }}.'
```

Comment:
(from the commit message)

> Grepping for python version won't actually raise an error when not found.
> This means it is only useful for debugging when looking at run logs.

Ah, no, that's not right. `grep` will exit 1 when it doesn't find a match, which will cause the job step to fail because [the default `shell` setting](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) includes bash's `-eo pipefail` options.

Reply:
Oh that makes sense now! I was trying to understand why [this was failing](https://github.com/nextstrain/cli/runs/5484946805?check_suite_focus=true#step:5:18) (before I used `bash -l {0}` so it was not activating the conda environment hence different version).

In that case, better to add the `grep` check back.